webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import 'babel-polyfill';
```

Comment:
IMHO this is not the right way to handle this, as it will actually load babel code into the browser, instead I think you should be using a babel transform plugin, e.g. like https://babeljs.io/docs/plugins/transform-runtime. this is consistent with other language enhancements we've added to babel/webpack. also it could be reused across the code base and not limited to this file.

Reply:
I will look into using transform-runtime for this.